upstream-community-operators/ext-postgres-operator/0.4.1/ext-postgres-operator.v0.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,160 @@
wn one database\r\n* Creates Kubernetes secret with postgres_uri in the same namespace as CR\r\n* Support for AWS RDS and Azure Database for PostgresSQL\r\n\r\n\r\n## Cloud specific configuration\r\n### AWS\r\nIn order for this operator to work correctly with AWS RDS, you need to set `POSTGRES_CLOUD_PROVIDER` to `AWS` either in \r\nthe ext-postgres-operator kubernetes secret or directly in the deployment manifest (`operator.yaml`).\r\n\r\n### Azure Database for PostgreSQL\r\nIn order for this operator to work correctly with Azure managed PostgreSQL database, two env variables needs to be provided for the operator:\r\n* `POSTGRES_CLOUD_PROVIDER` set to `Azure`\r\n* `POSTGRES_DEFAULT_DATABASE` set to your default database, i.e. `postgres`\r\n\r\n## Installation\r\nThis operator requires a Kubernetes Secret to be created in the same namespace as operator itself. \r\nSecret should contain these keys: POSTGRES\\_HOST, POSTGRES\\_USER, POSTGRES\\_PASS, POSTGRES\\_URI\\_ARGS, POSTGRES\\_CLOUD\\_PROVIDER, POSTGRES\\_DEFAULT\\_DATABASE.\r\nExample: \r\n\r\n```yaml\r\napiVersion: v1\r\nkind: Secret\r\nmetadata:\r\n  name: ext-postgres-operator\r\n  namespace: operators\r\ntype: Opaque\r\ndata:\r\n  POSTGRES_HOST: cG9zdGdyZXM=\r\n  POSTGRES_USER: cG9zdGdyZXM=\r\n  POSTGRES_PASS: YWRtaW4=\r\n  POSTGRES_URI_ARGS: IA==\r\n  POSTGRES_CLOUD_PROVIDER: QVdT\r\n  POSTGRES_DEFAULT_DATABASE: cG9zdGdyZXM=\r\n```\r\n\r\n## CRs\r\n\r\n### Postgres\r\n```yaml\r\napiVersion: db.movetokube.com\/v1alpha1\r\nkind: Postgres\r\nmetadata:\r\n  name: my-db\r\n  namespace: app\r\nspec:\r\n  database: test-db # Name of database created in PostgreSQL\r\n  dropOnDelete: false # Set to true if you want the operator to drop the database and role when this CR is deleted\r\n  masterRole: test-db-group\r\n  schemas: # List of schemas the operator should create in database\r\n  - stores\r\n  - customers\r\n```\r\n\r\nThis creates a database called `test-db` and a role `test-db-group` that is set as the owner of the database.\r\nReader and writer roles are also created. These roles have read and write permissions to all tables in the schemas created by the operator, if any.\r\n\r\n### PostgresUser\r\n```yaml\r\napiVersion: db.movetokube.com\/v1alpha1\r\nkind: PostgresUser\r\nmetadata:\r\n  name: my-db-user\r\n  namespace: app\r\nspec:\r\n  role: username\r\n  database: my-db # This references the Postgres CR\r\n  secretName: my-secret\r\n  privileges: OWNER # Can be OWNER\/READ\/WRITE\r\n```\r\n\r\nThis creates a user role `username-<hash>` and grants role `test-db-group`, `test-db-writer` or `test-db-reader` depending on `privileges` property. Its credentials are put in secret `my-secret-my-db-user`.\r\n\r\n`PostgresUser` needs to reference a `Postgres` in the same namespace.\r\n\r\nTwo `Postgres` referencing the same database can exist in more than one namespace. The last CR referencing a database will drop the group role and transfer database ownership to the role used by the operator."
```

Comment:
@hitman99, found unknown escape character: yaml: line 62 in your CSV. I tried to delete whole line and create new one with description with some random text and tests continue then. See, that description is not greed but black.

Reply:
I've tried to cleanup the description. It looks ok when I use the yaml file in "Preview your operator"

Reply:
The build still fails though, not sure where the exact issue is